### PR TITLE
feat(issue-69): show alert dialog before deleting connection

### PR DESCRIPTION
**Pull request by jskurt**
## Issue Link

[issue-69](https://github.com/kafelix496/coding-for-fun/issues/69)

## Notes

- I think I can't know success and fail. I think only know in use-installation.ts, Is it possible?

### CODE REVIEW

File: src/modules/github/components/connections/connections.tsx [hunk 1]
```diff
@@ -2,19 +2,77 @@
...
+    if (!dialogData.installationId) {
+      return
+    }
+    console.log('submit')
+    deleteInstallation(dialogData.installationId)
+      .then(() => {
+        console.log('success')
+        handleOpenChange(false)
+      })
+      .chatch(() => {
```
Comment (kafelix496) [inline src/modules/github/components/connections/connections.tsx:53]: chatch?

File: src/modules/github/components/connections/connections.tsx [hunk 2]
```diff
@@ -2,19 +2,77 @@
...
+        handleOpenChange(true)
```
Comment (kafelix496) [inline src/modules/github/components/connections/connections.tsx:54]: you don't need this. it's already opened

File: src/modules/github/components/connections/connections.tsx (hunk 1 above)
Comment (jskurt) [reply src/modules/github/components/connections/connections.tsx:53]: lol I already modified.........................................I really didn't know that WHAT THE ERROR.

File: src/modules/github/components/connections/connections.tsx [hunk 3]
```diff
@@ -44,14 +105,38 @@ export const Connections: FC = () => {
...
+        onActionClick={() => handleActionClick()}
+        title={dialogData.title}
+        children={
+          <>
+            <div>
+              Are you sure you want to delete this{' '}
+              {dialogData.installationOwner} connection?
+              <br />
+              You can not undo this action?
+            </div>
```
Comment (kafelix496) [inline src/modules/github/components/connections/connections.tsx:128]: Use the translate function for this. And remove `?` at the end of "You can not undo this action"

File: src/modules/github/components/connections/connections.tsx [hunk 4]
```diff
@@ -44,14 +105,38 @@ export const Connections: FC = () => {
...
+            {error !== '' && (
+              <Alert
+                variant="error"
+                title="Delete error"
```
Comment (kafelix496) [inline src/modules/github/components/connections/connections.tsx:132]: Use the translate function for this

File: src/dictionaries/en.ts [hunk 5]
```diff
@@ -9,10 +9,12 @@ export const dictionary = {
     APP_NAME: 'Coding For Fun',
     ALERT_DIALOG_DEFAULT_CANCEL_BUTTON: 'Cancel',
     ALERT_DIALOG_DEFAULT_CONTINUE_BUTTON: 'Continue',
-    ALERT_DIALOG_DEFAULT_SUBMIT_BUTTON: 'Submit'
+    ALERT_DIALOG_DEFAULT_SUBMIT_BUTTON: 'Submit',
+    ALERT_DIALOG_DEFAULT_CONFIRM_BUTTON: 'Confirm'
   },
   GITHUB: {
     NEW_CONNECTION_BUTTON: 'New Connection',
+    CONNECTION_DELETE_CONNECTION_TITLE: 'Delete Conneection',
```
Comment (kafelix496) [inline src/dictionaries/en.ts:17]: Conneection -> Connection

File: src/modules/github/components/connections/connections.tsx (hunk 3 above)
Comment (jskurt) [reply src/modules/github/components/connections/connections.tsx:128]: I resolved then how can I 'enter' between 'connection?' and 'You'? 🥲

Comment (kafelix496) [reply src/modules/github/components/connections/connections.tsx]: You can have two different translate

Comment (jskurt) [reply src/modules/github/components/connections/connections.tsx]: What the hell! Oh my god.

**Comment by jskurt** [comment]
rebased complete.

**Pull request merged**